src/fromager/wheels.py

```diff
@@ -117,7 +117,7 @@ def build_wheel(
 def default_build_wheel(
     ctx: context.WorkContext,
     build_env: BuildEnvironment,
-    extra_environ: dict,
+    extra_environ: dict[str, typing.Any],
```

Comment:
extra_environ is passed to functions that expect a `dict[str, str]` mapping.

```suggestion
    extra_environ: dict[str, str],
```

Reply:
I have added a TODO item to the ticket.

Reply:
Yes I was planning to have type alias for all our environ maps. I wasn't sure if `str` was fine so just kept it any for now